### PR TITLE
place action buttons in modal header, remove remaining footer space

### PR DESCRIPTION
**Pull request by aloupfor**

### CODE REVIEW

File: BackofficeBundle/Resources/public/less/smartadminPatches/modal.less [hunk 1]
```diff
@@ -8,8 +8,19 @@
         height: 90%;
         width: 90%;
 
+        .modal-header {
+            overflow: auto;
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/public/less/smartadminPatches/modal.less]: i think it's not a good idea to fix height and add an overflow, you can use instead a clearfix for manage the float

**Comment by aloupfor** [comment]
@itkg-alavieille @itkg-ngilain 
Is the following code in `modal.less` still used somewhere else or should I remove it?
```
.modal-footer {
    button.submit_form {
        float: left;
    }
}
```

File: BackofficeBundle/Resources/public/coffee/AdminFormView.coffee [hunk 2]
```diff
@@ -32,20 +32,20 @@ AdminFormView = OrchestraView.extend(
...
         if viewContext.deleteButton
-          originalButton = $('.submit_form', response)
-          button = originalButton.clone().attr('data-clone', originalButton.attr('id')).removeAttr('id')
-          footer = $('<div>')
-          .append(viewContext.renderTemplate('OpenOrchestraBackofficeBundle:BackOffice:Underscore/deleteButton', viewContext.options))
-          .prepend(button)
-          .html()
+          actionButtons.append(viewContext.renderTemplate('OpenOrchestraBackofficeBundle:BackOffice:Underscore/deleteButton', viewContext.options))
+        actionButtons = actionButtons.prepend(button)
+        .html()
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/public/coffee/AdminFormView.coffee]: you can remove the line return before ``.html()``, not sure it helps with the reading

File: BackofficeBundle/Resources/views/BackOffice/Underscore/orchestraModalView._tpl.twig [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
     <div class="modal-content">
-        <div class="modal-header">
+        <div class="modal-header clearfix">
             <button type="button" class="close" data-dismiss="modal">
                 <span aria-hidden="true">&times;</span>
                 <span class="sr-only">{{ ('open_orchestra_backoffice.modal.close') | trans() }}</span>
             </button>
+            <% if (typeof actionButtons != 'undefined') { %>
+            <%= actionButtons %>
+            <% } %>
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/views/BackOffice/Underscore/orchestraModalView._tpl.twig]: I'm not sure of what it looks in the browser: is the close button on the left side of the other buttons or on the right side ? The close button should be the last button on the right side.

Comment (aloupfor) [inline BackofficeBundle/Resources/views/BackOffice/Underscore/orchestraModalView._tpl.twig]: The close button is on the right side.
screenshot from 2015-11-16 10 18 10

**Comment by aloupfor** [comment]
@itkg-alavieille @itkg-ngilain update done

**Comment by itkg-ngilain** [comment]
:+1:

File: BackofficeBundle/Resources/public/less/smartadminPatches/modal.less [hunk 4]
```diff
@@ -8,8 +8,17 @@
         height: 90%;
         width: 90%;
 
+        .modal-header {
+            background: white;
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/public/less/smartadminPatches/modal.less]: why add a background color, by default it's not already white ?

**Comment by aloupfor** [comment]
@itkg-ngilain @itkg-alavieille back to fixed header height

File: BackofficeBundle/Resources/public/less/smartadminPatches/modal.less (hunk 4 above)
Comment (aloupfor) [inline BackofficeBundle/Resources/public/less/smartadminPatches/modal.less]: It's part of the mechanism to perform a flexible header height.

**Comment by itkg-alavieille** [comment]
:+1:

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**